index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// Key in annotations object that maps to an executor NPM module
+const ANNOTATION_EXECUTOR_TYPE = 'beta.screwdriver.cd/executor';
```

Comment:
Included a comment to clarify that the value is a placeholder until that issue is addressed. Thanks for the suggestion!